ERCS/eip-draft_open_ip_protocol.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Open IP Protocol
```

Comment:
Suggest using a more descriptive title: Open IP to remix NFTs

Reply:
Thank you very much for your thoughtful suggestion. After careful consideration and discussion among all co-authors, we have decided to stick with our original title. We hope you understand our perspective. Once again, we truly appreciate your valuable feedback.

Reply:
My phone hak